### PR TITLE
feat(sdk/python): add ci for auto push python distribution to pypi

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR is added ci for auto push python distribution to pypi

TODO:
- [x] remove sdk/python/LICENSE

### CODE REVIEW

File: .github/workflows/publish-to-pypi.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Push Python distributions to PyPI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+# on:
```
Comment (YikSanChan) [inline .github/workflows/publish-to-pypi.yaml:7]: do we need this?

**Review by YikSanChan — APPROVED** [review]
lgtm

File: .github/workflows/publish-to-pypi.yaml (hunk 1 above)
Comment (lianxmfor) [reply .github/workflows/publish-to-pypi.yaml:7]: First set the trigger condition to debug when PR occurs, then use the commented code (tag trigger) when it is OK

**Review by wfxr — APPROVED** [review]
lgtm

**Pull request merged**